src/htmlreporter.ts

```diff
@@ -123,6 +123,58 @@ class HTMLReporter {
     return report;
   }
 
+  public renderTestSuiteInfo(parent: XMLElement, suite: TestResult) {
+    const suiteInfo2 = parent.ele("div", { class: "suite-info" });
+    // Suite Path
+    suiteInfo2.ele("div", { class: "suite-path" }, suite.testFilePath);
+    // Suite execution time
+    const executionTime2 = (suite.perfStats.end - suite.perfStats.start) / 1000;
+    suiteInfo2.ele(
+      "div",
+      {
+        class: `suite-time${
+          executionTime2 >
+          (this.getConfigValue("executionTimeWarningThreshold") as number)
+            ? " warn"
+            : ""
+        }`,
+      },
+      `${executionTime2}s`
+    );
```

Comment:
Any reason to why the constant names are ending with 2?

Reply:
Oops
Forgot to change them after refactor
Fixed, thx!

Reply:
@Hargne